async function main() {
    // do something...
}
main().catch((error) => {
    console.error(error);
    process.exitCode = 1;
});